.github/workflows/main.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       matrix:
         otp_version: ['24', '23', '22', '21']
-        os: [ubuntu-latest]
```

Comment:
Upon merge this'll get a conflict and will be resolved. It's possible we'll have to stick with 20.04, if we wanna support pre-24, since Hex only builds 24+ for Erlang/OTP, in Ubuntu 22.04)

Reply:
I'm partial to supporting 24+. So always the last 3 releases.

Reply:
You mean previous -3? (asking because 27 is already out 😄)

Reply:
hah, my math failed me. But yea, unless there is something that came with OTP-25 that lets us drop code required just to support OTP-24 lets go with -4, so 24-27.

Reply:
For the time being 26-27 is not possible (at least tests fail) because of https://github.com/elli-lib/elli/pull/114#issuecomment-1656698857. But I can already adjust CI and `rebar.config` to consider 24 as minimum.